core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.component.ts

```diff
@@ -97,37 +97,41 @@ interface ActiveContributionDict {
 })
 export class QuestionSuggestionReviewModalComponent
   extends ConfirmOrCancelModal implements OnInit {
-  @Input() reviewable: boolean;
-  @Input() suggestionId: string;
-  @Input() misconceptionsBySkill: MisconceptionSkillMap;
-
   @Output() editSuggestionEmitter = (
     new EventEmitter<QuestionSuggestionModalValue>());
 
-  reviewMessage: string;
-  questionStateData: State;
-  questionId: string;
-  canEditQuestion: boolean;
-  skillDifficultyLabel: string;
-  skillRubricExplanations: string | string[];
-  suggestionIsRejected: boolean;
-  validationError: string;
+  // These properties below are initialized using Angular lifecycle hooks
+  // where we need to do non-null assertion. For more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() reviewable!: boolean;
+  @Input() suggestionId!: string;
+  @Input() misconceptionsBySkill!: MisconceptionSkillMap;
+  reviewMessage!: string;
+  questionStateData!: State;
+  // Question id is null when it is not used in the backend.
```

Comment:
```suggestion
  // Question ID is null when it is not used in the backend.
```

Also, what do you mean by "not used in the backend"?

Reply:
Done